code/go/0chain.net/miner/protocol_block_integration_tests.go

```diff
@@ -210,6 +211,23 @@ func isIgnoringGenerateBlock(rNum int64) bool {
 	return cfg != nil && cfg.OnRound == rNum && nodeType == generator && typeRank == 1
 }
 
+func (mc *Chain) GenerateBuiltInTxns(ctx context.Context, lfb, b *block.Block) ([]*transaction.Transaction, int, error) {
+	DefaultHardforkConfig := crpc.Client().State().Hardfork
+
+	txns, cost, err := mc.buildInTxns(ctx, lfb, b)
+	if DefaultHardforkConfig != nil && b.Round == 1 {
+		DefaultHardfork := DefaultHardforkConfig.Name
+		addHardforkTxn, err := mc.createHardforkTxn(b, DefaultHardfork)
```

Comment:
it's not correct place to add hardfork, it is run on every block that miner produce. also only owner can add hardforks, so you need to run a script that will send transaction to miner using the owner wallet